schema/core.md

```diff
@@ -607,57 +609,66 @@ are used for a particular study.
 Note: `hasPart` and `isPartOf` are generally used for records that are grouped
 together or records that belong to other groupings. For example, `hasPart` may
 be used to indicate a record that consists of one or more additional
-CreativeWork (this is similar to a HydroShare collection). In contrast,
+`CreativeWork` (this is similar to a HydroShare collection). In contrast,
 `isPartOf` is used to indicate catalog records that reference a designated
-CreativeWork. As such, these are not demonstrated in the example below.
+`CreativeWork`. 
 
+The example below shows a simple schema for a `CreativeWork` that links to another
+`CreativeWork`.
 
 ``` json
 {
+  "@type": "CreativeWork",
+  "name": "Collection of Sample Datasets for IGUIDE",
+  "description": "A collection of sample datasets used by the I-GUIDE team",
+  "identifier": "https://my-unique-url.com/data-collection",
   "hasPart": {
-    "@type": "Dataset",
+    "@type": "CreativeWork",
     "name": "IGUIDE Shapefile Testing Resource",
-    "description": "Test HydroShare resource for I-GUIDE",
-    "url": "https://www.hydroshare.org/resource/9d413b9d57824a79b8239a5f7c4fdf51/data/contents/HUC6_Harvey_TxLaMsAr.shp?zipped=true&aggregation=true"
+    "description": "Test HydroShare resource for I-GUIDE - Shapefile",
+    "identifier": "https://my-unique-url.com/shapefile"
```

Comment:
Again - use a URL-encoded identifier that actually looks like an identifier.

Reply:
Suggest that we remove the `url` property for now (as Irene has already done) and replace `identifier` with the following:

```
   ...
   "identifier": "<internal-record-guid>"
```

and add text to the page that explains that the `<internal-record-guid>` is a programmatically generated record locator used internally.